ganttproject/src/main/java/net/sourceforge/ganttproject/calendar/MultiDatePicker.java

```diff
@@ -0,0 +1,105 @@
+package net.sourceforge.ganttproject.calendar;
```

Comment:
We need a copyright notice here. This code came from StackOverflow, right?

Reply:
Done